backend/handler/db.js

```diff
@@ -268,6 +275,12 @@
 		return collection.update(query, data, options);
 	};
 
+	Handler.updateMany = async function (database, colName, query, data, upsert = false) {
```

Comment:
you added an `updateMany` when `update` function is still here. 

If we want to stop using update then we should outright remove the function so it doesn't get confusing.